services/scoring-engine/src/engine.ts

```
@@ -0,0 +1,658 @@
+        break;
+
+      case 'any':
+        // At least one assertion must pass
+        passed = results.some((r) => r.passed);
+        overallScore = passed ? 1.0 : 0.0;
+        break;
+
+      case 'weighted_average':
+      default:
+        // Weighted average of assertion scores
+        const totalWeight = assertions.reduce((sum, a) => sum + (a.weight ?? 1), 0);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)